src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -644,22 +644,22 @@ typedef VOID (*RtcOnPictureLoss)(UINT64);
  *
  * Reference: https://www.w3.org/TR/webrtc/#dom-rtcdatachannel-onmessage
  */
-typedef VOID (*RtcOnMessage)(UINT64, BOOL, PBYTE, UINT32);
+struct __RtcDataChannel;
```

Comment:
Do we really need this forward declaration? I see we have this elsewhere in the header  LN 664 but I believe this should not be necessary as the declaration of the structs fkr RtcDataChannel and RtcDataChannelInit should be moved before the function declarations

Reply:
It was there and I don't know exactly what to do. Can you suggest a change?

Reply:
Oh sure, __RtcDataChannel is declared below the function calls that uses it, hence the need for the forward declaration. You could move the declaration above the callback declaration.

This is really a minor change so you can chose to ignore it - no worries

Reply:
> You could move the declaration above the callback declaration.
I did in 1666077